src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -9,6 +11,12 @@ export const INTERNET_IDENTITY_CANISTER_ID = LOCAL
 	? import.meta.env.VITE_LOCAL_INTERNET_IDENTITY_CANISTER_ID
 	: undefined;
 
+export const INTERNET_IDENTITY_ORIGIN = LOCAL
```

Comment:
There is no II on staging  actually - i.e. here staging === mainnet

Reply:
Done!